app/public/js/cookieMessage.js

```diff
@@ -4,7 +4,7 @@
   var document = global.document;
 
   function setCookie(name, value, options = {}) {
-    let cookieString = `${name}=${value}; path=/`;
+    var cookieString = `${name}=${value}; path=/`;
```

Comment:
the template literal will need to change too as they are not supported in Internet Explorer

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Browser_compatibility

Reply:
Fixed